File: interface/src/main/java/com/kgc/pinyougou/service/GoodsDescService.java
package com.kgc.pinyougou.service;

import com.kgc.pinyougou.domain.GoodsDesc;
import com.kgc.pinyougou.utils.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface GoodsDescService {

	/**
	 * @Author Admin
	 * @description:  //TODO
	 * @param:
	 * @param pageNum   分页页数
	 * @param pageSize  分页记录数
	 * @return: com.kgc.pinyougou.utils.PageResult
	 * @time: 2020/4/25 22:11
	 */
	PageResult findAllGoodsDesc(Integer pageNum, Integer pageSize, GoodsDesc goodsDesc);

	Integer insertGoodsDesc(GoodsDesc goodsDesc);

	GoodsDesc findOne(Long id);

	Integer updateGoodsDesc(GoodsDesc goodsDesc);

	Integer deleteGoodsDesc(Long [] ids);
}

File: pinyougou-manager-web/src/main/resources/static/js/service/goodsDescService.js
//服务层
app.service('goodsDescService',function($http){

    this.findAllGoodsDesc=function (page,size,searchEntity) {
        return $http.post("../goodsDesc/findAllGoodsDesc?page="+page+"&size="+size,searchEntity)
    }

    this.findOne=function (id) {
        return $http.get("../goodsDesc/findOne?id="+id)
    }

    this.del=function (ids) {
        return $http.post("../goodsDesc/delGoodsDesc",ids)
    }

    this.add=function (searchEntity) {
        return $http.post("../goodsDesc/addGoodsDesc",searchEntity)
    }

    this.update=function (searchEntity) {
        return $http.post("../goodsDesc/updateGoodsDesc",searchEntity)
    }
});
